atlassian_jwt_auth/frameworks/wsgi/tests/test_wsgi.py

```diff
@@ -63,19 +63,42 @@ def test_request_with_valid_token_is_allowed(self):
             'client-app/key01', self._private_key_pem
         )
         body, resp_info, environ = self.send_request(token=token)
-        assert resp_info['status'] == '200 OK'
-        assert 'ATL_ASAP_CLAIMS' in environ
+        self.assertEqual(resp_info['status'], '200 OK')
+        self.assertIn('ATL_ASAP_CLAIMS', environ)
+
+    @unittest.expectedFailure
```

Comment:
@mark-adams I haven't been able to determine why the wsgi middleware doesn't behave like the other middleware with regards to respecting the `ASAP_CHECK_JTI_UNIQUENESS` setting.

Reply:
I ended up working out that I needed to persist the wsgi application when sending more than one request to fix the test.